src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/StatusChangeConfirmation.cshtml

```diff
@@ -90,19 +91,23 @@
                 </thead>
                 <tbody>
                     <tr>
-                        <td>Name</td>
-                        <td class="bold">@Model.Data.ApprenticeName</td>
-                    </tr>
-                    <tr>
-                        <td>Unique learner number</td>
-                        <td class="bold">@Model.Data.ApprenticeULN</td>
+                        <th scope="row" class="bold">Name</th>
```

Comment:
The view was already using the `bold` class on the 'td', but ideally, the font-weight should be set in the css on the `td`/`th`, rather than explicitly supplying a bold class. That's so that we can change the layout by changing the css, rather than the markup.

Reply:
The  bold class sets the font weight in the css. Not every td or th may want to be displayed the same hence the inclusion of the correct CSS class

Reply:
It's no worse than is there currently, so it doesn't warrant not approving this PR, but I think you've missed my point.

Reply:
Bold is now applied using the new `details-list-new` class.